scale-tidb-using-tiup.md

```diff
@@ -22,7 +22,7 @@ For example, if the original topology of the cluster is as follows:
 | 10.0.1.1 | TiKV |
```

Comment:
Please also update line 11.

Reply:
Done in https://github.com/pingcap/docs/pull/3713/commits/64ca17f1bd695d6d99e0bec0e1e7405dcedce8be